qt/aqt/reviewer.py

```diff
@@ -152,8 +152,8 @@ def __init__(self, mw: AnkiQt) -> None:
         self.previous_card: Card | None = None
         self._answeredIds: list[CardId] = []
         self._recordedAudio: str | None = None
-        self.typeCorrect: str = None  # web init happens before this is set
-        self.state: Literal["question", "answer", "transition", None] = None
+        self.typeCorrect: str | None = None  # web init happens before this is set
+        self.state: Literal["question", "answer", "transition", None] | None = None
```

Comment:
This looks redundant. Was it a required change? I'll shift it outside the literal for now, in case it was required to satisfy a type checker, as either works for me.

```suggestion
        self.state: Literal["question", "answer", "transition"] | None = None
```

Reply:
Good catch! My change seems indeed to have been redundant. Thanks for cleaning it up.